pycaret/internal/preprocess/time_series/forecasting/preprocessor.py

```diff
@@ -187,7 +189,9 @@ def _scaling(
         if exogenous_present == TSExogenousPresent.YES and scale_exogenous is not None:
             self._add_scaling_steps(scale=scale_exogenous, target=False)
 
-    def _add_scaling_steps(self, scale: str, target: bool = True):
+    def _add_scaling_steps(
+        self, scale: Optional[Union[str, int, float]], target: bool = True
```

Comment:
This should not be int and float and not Optional either since we check for it before passing to this function.

https://github.com/pycaret/pycaret/blob/master/pycaret/time_series/forecasting/oop.py#L1250-L1255

https://github.com/pycaret/pycaret/blob/master/pycaret/internal/preprocess/time_series/forecasting/preprocessor.py#L181-L189

Reply:
@ngupta23 
Regarding this I have fix based on comments. 

https://github.com/pycaret/pycaret/pull/3081/commits/20dd505f5c6a4be3200526069ceac39f19367991

Thank you.